a/appstream/manifest.x86_64.jsonc

```diff
@@ -27,19 +27,29 @@
 			"depends": [
 				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
 				"soname(libc.so.6(x86_64))",
+				"soname(libcairo.so.2(x86_64))",
 				"soname(libcurl.so.4(x86_64))",
+				"soname(libfontconfig.so.1(x86_64))",
+				"soname(libfreetype.so.6(x86_64))",
 				"soname(libfyaml.so.0(x86_64))",
+				"soname(libgdk_pixbuf-2.0.so.0(x86_64))",
 				"soname(libgio-2.0.so.0(x86_64))",
 				"soname(libglib-2.0.so.0(x86_64))",
 				"soname(libgobject-2.0.so.0(x86_64))",
+				"soname(libpango-1.0.so.0(x86_64))",
+				"soname(librsvg-2.so.2(x86_64))",
 				"soname(libsystemd.so.0(x86_64))",
 				"soname(libxml2.so.16(x86_64))",
 				"soname(libxmlb.so.2(x86_64))",
 				"soname(libzstd.so.1(x86_64))"
 			],
 			"files": [
 				"/usr/bin/appstreamcli",
+				"/usr/lib/appstream/appstreamcli-compose",
 				"/usr/lib/girepository-1.0/AppStream-1.0.typelib",
+				"/usr/lib/girepository-1.0/AppStreamCompose-1.0.typelib",
+				"/usr/lib/libappstream-compose.so.0",
+				"/usr/lib/libappstream-compose.so.1.1.1",
```

Comment:
Since appstream-compose adds new rundeps I think I'd prefer if we split it out similar to how Fedora does